langevals/scripts/generate_evaluators_ts.py

```
@@ -50,6 +50,118 @@ def stringify_field_types(field_types):
+                    or get_origin(f.annotation) == Optional
+                    or (
+                        get_origin(f.annotation) == Union
+                        and type(None) in get_args(f.annotation)
+                    ),
+                }
+                for field_name, f in field.model_fields.items()
+            },
+            definitions,
+        )
+        return f"z.object({{{inner}}})"
+    if field == str:
```

Ruff (0.15.10)

[error] Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)